springer-basic-author-date.csl

```diff
@@ -121,10 +113,16 @@
              In: Smith J (ed) The rise of modern genomics, 3rd edn.
              Wiley, New York, pp 230-257 -->
         <if type="chapter paper-conference" match="any">
-          <text macro="editor"/>
-          <group prefix=" " delimiter=", " suffix=". ">
-            <text variable="container-title"/>
-            <text macro="edition"/>
+          <text term="in" text-case="capitalize-first" suffix=": "/>
+          <group delimiter=" " suffix=". ">
+            <names variable="editor">
+              <name sort-separator=" " initialize-with="" name-as-sort-order="all" delimiter=", " delimiter-precedes-last="always"/>
+              <label form="short" strip-periods="true" prefix=" (" suffix=")"/>
+            </names>
+            <group delimiter=", ">
+              <text variable="container-title" form="short"/>
```

Comment:
I don't think we want the form="short" here -- in Zotero/Mendeley that will force journal-style abbreviations on the book title, which is almost certainly wrong (it definitely is for the example above).